File: Warmup-2/arrayCount9/ArrayCount9.java
package arrayCount9;

public class ArrayCount9 {
    //Retorne o número de 9's que possui o Array.

    public int arrayCount9(int[] nums) {
        int cont = 0;
        for (int n :
                nums) {
            if (n == 9) cont++;
        }
        return cont;
    }
}

File: Python/Warmup-2/array_count9.py
"""
Dado um array numérico, retorne o número de "9" que ele possui.
"""


def array_count9(nums):
    return nums.count(9)


# UTILIZANDO UM FOR COM ITERADOR DE ELEMENTOS
def array_count9_2(nums):
    count = 0
    # Standard loop to look at each value
    for num in nums:
        if num == 9:
            count = count + 1

    return count


print(array_count9([1, 2, 9, 9]))
print(array_count9_2([9, 9, 9, 9]))
